tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -171,3 +171,54 @@ Feature: collaboration (wopi)
       | app-endpoint                                     |
       | /app/open?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open?file_id=<<FILEID>>                     |
+
+
+  Scenario Outline: public user open file with .odt extension
+    Given user "Alice" has uploaded file "filesForUpload/simple.odt" to "simple.odt"
+    And we save it into "FILEID"
+    And user "Alice" has created the following resource link share:
+      | resource        | simple.odt |
+      | space           | Personal   |
+      | permissionsRole | view       |
+      | password        | %public%   |
+    When the public sends HTTP method "POST" to URL "<app-endpoint>" with password "%public%"
```

Comment:
IMO `When the public sends ...` seems little confusing. Will it better if we put like `When the public user sends ....`?

Reply:
Just following other similar steps for the public users.